rio/src/RIO/Process.hs

```diff
@@ -309,8 +316,8 @@ mkDefaultProcessContext =
 
 -- | Modify the environment variables of a 'ProcessContext'.
 --
--- This will keep other settings unchanged, in particular the working
--- directory.
+-- This will keep other settings that do not depend on the environment variables
```

Comment:
I don't understand what this change is adding. The new language sounds more confusing to me. Is this relevant to this PR?